Polyhedron/demo/Polyhedron/Plugins/Display/Display_property_plugin.cpp

```diff
@@ -443,9 +443,6 @@ class DisplayPropertyPlugin :
     connect(dock_widget->deleteButton, &QPushButton::clicked,
             this, &DisplayPropertyPlugin::delete_group);
 
-    connect(dock_widget->resetButton, &QPushButton::pressed,
-            this, &DisplayPropertyPlugin::resetRampExtremas);
```

Comment:
That does not seem related to spin boxes.

Reply:
It is not, the commit explains that change. I spotted a bug while testing the spin box replacement, and "fixed " it here.